Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5308,6 +5309,39 @@ def com_google_fonts_check_stylisticset_description(ttFont):
         yield PASS, "OK"
 
 
+@check(
+    id="com.google.fonts/check/os2/fsselectionbit7",
+    rationale="""
+    All fonts should have OS/2.fsSelection bit 7 (USE_TYPO_METRICS) set \
+    so that they use typo vertical metrics instead of Win vertical metrics.
```

Comment:
Do not break lines in the rationale. Font Bakery will do line-breaking for us based on the needs of each specific type of reporter.

Also, please indent that block with 4 additional spaces (we have mandatory indentation of a total of 8 spaces on rationale strings)

Reply:
https://github.com/googlefonts/fontbakery/pull/3314/commits/beae3f971f0180bc08ac4aac767ed58e0ea77c98